autospec/files.py

```diff
@@ -184,144 +317,24 @@ def push_file(self, filename):
         part = re.compile(r"^/usr/lib/systemd/system/.+\.target\.wants/.+")
         if part.search(filename) and 'update-triggers.target.wants' not in filename:
             self.push_package_file(filename, "autostart")
-            self.excludes.append(filename)
-
-        # extras
-        if filename in self.extras:
-            self.push_package_file(filename, "extras")
-            self.excludes.append(filename)
-        if filename in self.dev_extras:
-            self.push_package_file(filename, "dev")
-            self.excludes.append(filename)
-
-        if filename in self.setuid:
-            newfn = "%attr(4755, root, root) " + filename
-            self.push_package_file(newfn, "setuid")
-            self.excludes.append(filename)
+            self.push_package_file("%exclude " + filename)
```

Comment:
I almost feel like we'd like an excludes flag for a push_package_file so the syntax for this is consolidated. I need to think about it more.